themes/startwords/assets/scss/main.scss

```diff
@@ -0,0 +1,4 @@
+// Startwords stylesheet
+@import 'global';
+@import 'a11y';
+@import 'print';
```

Comment:
Do we want this included in the main scss?  I had imagined it as a separate include so we can specify media, but I guess we can scope to print media within the print.scss file?

Reply:
we can scope it, but you're right - this reminded me that we probably do want it in a separate file so that we only include it when the user prints the page, to save space. there's a way to do this in the `link` tag, iirc.

Reply:
Yes, that's what I had in mind.